marklogic-data-hub-central/ui/e2e/cypress/integration/scenarios.queries.tsx

```diff
@@ -74,6 +81,51 @@ describe('save/manage queries scenarios', () => {
         queryComponent.getDeleteQueryYesButton().click({force: true})
     });
 
+});
+
+describe('save/manage queries scenarios, operator role', () => {
+
+    beforeEach(() => {
+        cy.visit('/');
+        cy.contains('MarkLogic Data Hub');
+        cy.loginAsOperator();
+        cy.wait(500);
+        homePage.getBrowseEntities().click();
+        cy.wait(1000);
+        browsePage.getFacetView();
+        browsePage.selectEntity('All Entities');
+    });
+
+    it('apply facet search,open save modal and save queries', () => {
```

Comment:
I would recommend trying to achieve this using RTL as well. The story is for export data and this test seems to be for search/save queries. Exporting data could be an e2e scenario, although I would recommend trying to use RTL first.